README.md

```diff
@@ -0,0 +1,8 @@
+Mailberry Forms library to use together with https://mailberry.ai form builder 
+
+In order to use this library in mode local after to make git clone and npm install, follow this steps:
+Consider my-app as the project name and my-library as the library name.
+
+1. <code>npm link 'path_to_my-app/node_modules/react'</code> from my-library, to use the same react version and avoid lint errors.

```

Comment:
please add documentation for final user at the top. This piece of doc that is aimed for the lib devs (us) can be below after a divider line -----

Reply:
add a snippet of code as an example for the final users too.